Pod/Classes/TMPodspec.h

```diff
@@ -18,9 +18,19 @@
 @property (nonatomic, readonly) NSString *version;
 
 /**
- *  @see http://guides.cocoapods.org/syntax/podspec.html#summary
+ *  @see http://guides.cocoapods.org/syntax/podspec.html#authors
  */
-@property (nonatomic, readonly) NSString *summary;
+@property (nonatomic, readonly) NSArray /* <TMAuthor> */ *authors;
```

Comment:
About as pedantic as can be, but I'd prefer if the spaces were removed to make it ``NSArray/*<TMAuthor>*/ *authors;`, just because it'd be slightly closer to how it'd read in a language like Swift or Java with actual generics (`Array<Author>`)